### PR TITLE
Updated Plugins documentation.

### PR DESCRIPTION
**Pull request by targed**
I reverted the links in the README not working, fixed some formatting issues, and updated the plugin information.

### CODE REVIEW

File: docs/docs/plugins/README.md [hunk 1]
```diff
@@ -4,13 +4,11 @@
 - [Plugins List](/plugins/list.md)
 - [Plugins Technical Details](/plugins/details.md)
 
-:::note
+# Note
```
Comment (andrewm4894) [inline docs/docs/plugins/README.md:7]: The :::note is a special docusaurus thing ("admonition") to make it look nice 

image

https://docusaurus.io/docs/markdown-features/admonitions

So i'd suggest leave them alone

File: docs/docs/plugins/getting-started.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Getting Started
 
+Check out the **unofficial** community plugin list available [here](https://github.com/targed/GPT-Plugins) to see what plugins are available.
```
Comment (andrewm4894) [inline docs/docs/plugins/getting-started.md:3]: this should probably not be at the top of the page

Comment (andrewm4894) [reply docs/docs/plugins/getting-started.md:3]: actually should not be on this page most probably i think

File: docs/docs/plugins/list.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 Below is a list of some available plugins.
 
+An **unofficial** community plugin list is available [here](https://github.com/targed/GPT-Plugins)
```
Comment (andrewm4894) [inline docs/docs/plugins/list.md:5]: also should not be at top of page.

**Comment by andrewm4894** [comment]
Maybe we should add a "# Useful resources" section at and we could add relevant links in there? 

So something like 

```
# Official Plugins
# Community Plugins
# Useful Resources
```

We could then add your link for https://github.com/targed/GPT-Plugins in there maybe?

**Comment by targed** [comment]
Good idea. I will fix it and update it. Give me a moment.

**Comment by targed** [comment]
Sorry about this. I wasn't thinking when I originally made the PR, and thus it was not up to par. I will close this and submit one at a future date with major updates to the docs instead of trivial stuff like this. My bad. Sorry for wasting your time.

**Pull request closed without merging**